cv32e40s/regress/cv32e40s_full.yaml

```diff
@@ -428,4 +428,12 @@ tests:
     builds: [ uvmt_cv32e40s_b_ext_abs, uvmt_cv32e40s_b_ext_all]
     dir: cv32e40s/sim/uvmt
     cmd: make test TEST=b_ext_test
+
+#TODO:MT Comment back in when Zc is enabled in the core
```

Comment:
should this only be run with pma cfg 1?

Reply:
I intend to revisit this when the regression is commented back in.